### ISSUE TITLE
techdocs-cli serve WSOD (v0.3.5) when mkdocs.yml contains edit_uri

### ISSUE THREAD

**Issue by bcotton** [backstage/techdocs-cli#49]
First time trying techdocs-cli on an existing mkdocs project. Installed 0.3.5 of techdocs-cli

The mkdocs project renders and serves fine with `techdocs-cli serve:mkdocs` However with the just the 'serve' command it starts to render the Backstage UI in the browser, then errors:

JSConsole shows:

```
Error: Missing required config value at 'integrations.github'
    at b.getConfigArray (index.esm.js:104)
    at index.esm.js:240
    at index.esm.js:973
    at Array.forEach (<anonymous>)
    at J (index.esm.js:972)
    at index.esm.js:1104
    at ro (react-dom.production.min.js:211)
    at gu (react-dom.production.min.js:257)
    at e.unstable_runWithPriority (scheduler.production.min.js:19)
    at Wl (react-dom.production.min.js:122)
```

The rendered js looks like:
```
const addGitFeedbackLink = (configApi) => {
  return (dom) => {
    const sourceAnchor = dom.querySelector('[title="Edit this page"]');
    if (!sourceAnchor || !sourceAnchor.href) {
      return dom;
    }
    let gitHost = "";
    const sourceURL = new URL(sourceAnchor.href);
    const githubHosts = configApi.getConfigArray("integrations.github").map((integration) => integration.data.host);
    const gitlabHosts = configApi.getConfigArray("integrations.gitlab").map((integration) => integration.data.host);
    if (githubHosts.includes(sourceURL.hostname) || sourceURL.origin.includes("github")) {
      gitHost = "github";
    } else if (gitlabHosts.includes(sourceURL.hostname) || sourceURL.origin.includes("gitlab")) {
      gitHost = "gitlab";
    } else {
      return dom;
    }
    const title = dom.querySelector("article>h1").childNodes[0].textContent;
    const issueTitle = encodeURIComponent(`Documentation Feedback: ${title}`);
    const issueDesc = encodeURIComponent(`Page source:
${sourceAnchor.href}

Feedback:`);
    const repoPath = sourceURL.pathname.split("/").slice(0, 3).join("/");
    const feedbackLink = sourceAnchor.cloneNode();
    switch (gitHost) {
      case "gitlab":
        feedbackLink.href = `${sourceURL.origin}${repoPath}/issues/new?issue[title]=${issueTitle}&issue[description]=${issueDesc}`;
        break;
      case "github":
        feedbackLink.href = `${sourceURL.origin}${repoPath}/issues/new?title=${issueTitle}&body=${issueDesc}`;
        break;
      default:
        return dom;
    }
    ReactDOM.render(React.createElement(FeedbackOutlinedIcon), feedbackLink);
    feedbackLink.style.paddingLeft = "5px";
    feedbackLink.title = "Leave feedback for this page";
    feedbackLink.id = "git-feedback-link";
    sourceAnchor == null ? void 0 : sourceAnchor.insertAdjacentElement("beforebegin", feedbackLink);
    return dom;
  };
```

node --version
v16.0.0
OSX Big Sur
Latest Chrome

Private gitlab repo, also

**Comment by bcotton** [comment]
Chased it down to the presence of an `edit_uri` in the mkdocs.yml file. When I remove that item, it renders fine.

**Comment by emmaindal** [comment]
Hey @bcotton we've just released a new version 0.3.6, which includes a potential fix to this issue in the techdocs plugin (PR merged here https://github.com/backstage/backstage/pull/5461/files). Could you try run the new version and see if that solves your issue? 

Otherwise, please re-open this!
